sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -21,6 +21,7 @@ DEPEND="gnuefi? ( >=sys-boot/gnu-efi-3.0.2 )
 DOCS=(README.txt)
 PATCHES=("${FILESDIR}/makefile.patch")
 UDK_WORKSPACE="${T}/udk"
+LIBDIR="/usr/lib"
```

Comment:
Hmm, you define this variable, but don't use it anywhere. Does the upstream makefile need it? Why not make this into `get_libdir` correctly, that should work with 17.1 profiles?

Reply:
That's a honest tired mistake.